### PR TITLE
Create math.js

### PR DESCRIPTION
**Pull request by dfarr**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/foo/pull/3" target="_blank">ReviewNinja

File: math.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+
+  add: function(a, b) {
+    return a + b;
+  },
+  
+  sub: function(a, b) {
+    return a - b
```
Comment (dfarr) [inline math.js]: Missing semicolon, please !fix

Comment (dfarr) [inline math.js]: !resolved

**Comment by dfarr** [comment]
:+1:

**Comment by dfarr** [comment]
!unstar

**Pull request merged**